tests/e2e/performace/performance.spec.js

```diff
@@ -0,0 +1,39 @@
+import { test, expect } from 'playwright-test-coverage'
+
+import singleCollection from './__mocks__/single_collection.json'
+
+test.describe('Performance Benchmarking', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.route('**/*.{png,jpg,jpeg}', (route) => route.abort())
+    await page.route(/collections/, async (route) => {
+      await route.fulfill({
+        json: singleCollection.body,
+        headers: singleCollection.headers
+      })
+    })
+  })
+
+  test('Search page load time is less than 1 second', async ({ page, browserName }) => {
```

Comment:
We should update the test string since it looks like we are comparing to 3 seconds here

Reply:
Updated.